month/months.js

```diff
@@ -0,0 +1,102 @@
+const today = new Date(Date.now())
+let currentMonth = today.getMonth()
+let currentYear = today.getFullYear()
+
+document.querySelector("#months-next").addEventListener('click', () => {
+    incrementMonth(1)
+    fillMonth(currentYear, currentMonth)
+})
+document.querySelector("#months-prev").addEventListener('click', () => {
+    incrementMonth(-1)
+    fillMonth(currentYear, currentMonth)
+})
+
+function updateLabel(){
+    const monthViewLabelMonth = document.querySelector("#monthView-label-month")
+    const monthViewLabelYear = document.querySelector("#monthView-label-year")
+    monthViewLabelMonth.innerHTML = new Date(currentYear, currentMonth).toLocaleDateString('us-US', {month: 'long'})
+    monthViewLabelYear.innerHTML = currentYear
+}
+
+function incrementMonth(value){
+    currentMonth += value
+    if(currentMonth > 11){
+        currentMonth = currentMonth%12
+        currentYear++
+    }
+    if(currentMonth < 0){
+        currentMonth += 12
+        currentYear--
+    }
+}
+
+function putInContainer(element){
```

Comment:
does this need to be a separate function? It doesn't really do anything special, is used in one place and has super generic name.
I would just have it in generateCell immediately